JapaneseFilter/sections/specific.txt

```diff
@@ -40,6 +40,7 @@ hpplus.jp,tv-tokyo.co.jp,financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.j
 !
 !
 pokemongo-kouryakumatome.net##.Article__content > div[align="center"] > br
+oshiete.goo.ne.jp###q-and-a > div[style="margin-bottom: 20px; text-align:center;"]
```

Comment:
I guess `oshiete.goo.ne.jp##div[style="margin-bottom: 20px; text-align:center;"]` is ok.